aat/src/resources/features/F-023/S-006.td.json

```diff
@@ -0,0 +1,33 @@
+{
+  "title": "Must return a negative response on malformed Case Type ID",
+  "_guid_": "S-006",
+  "_extends_": "F-023_Test_Data_Base",
+  "specs": [
+    "an active profile in CCD - wants to see a list of valid documents they can print",
+    "contains a malformed caseTypeID"
+  ],
+
+  "request": {
+    "pathVariables": {
+      "ctid": "$%£@^&*()_+",
+      "cid": "1111222233334444"
+    }
+    },
+
+  "expectedResponse": {
+    "responseCode": 400,
+    "responseMessage": "Bad Request",
```

Comment:
Please extent the common 400 response object. That will bring the 2 line shere and the headers below.

Reply:
done